tests/test_stations.py

```diff
@@ -1,18 +1,59 @@
 # -*- coding: utf-8 -*-
-
+import os
 import unittest
 
+from openkongqi.conf import config_from_object
 from openkongqi.utils import get_uuid
 
 
+here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Add a comment here to explain why this whole configuration needs to happen before importing `get_station_map`